packages/tabsheet/src/vaadin-tabsheet.js

```diff
@@ -157,7 +157,7 @@ class TabSheet extends ControllerMixin(DelegateStateMixin(ElementMixin(ThemableM
 
   /** @override */
   static get delegateProps() {
-    return ['selected'];
+    return ['selected', 'theme'];
```

Comment:
Consider using `delegateAttrs` for setting `theme` because it's an attribute and not a property.

Reply:
Done